### ISSUE TITLE
Question about Evaluating

### ISSUE THREAD

**Issue by zeratul233** [sukjunhwang/IFC#12]
How can I obtain the FPS and AP data for the Neural Network on the validation set?

**Comment by sukjunhwang** [comment]
FPS is measured by total frames in a validation dataset, divided by the total seconds it took to process the dataset.
Total time gets scripted when an inference is completed, so you can use the time to measure the FPS.
For AP, you should upload the result file on Codalab evaluation server, which you can check on the YouTube-VIS challenge website.
